artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -909,10 +943,25 @@ components:
         application/json:
           schema:
             $ref: "#/components/schemas/ErrorInfo"
-          example:
-            status: 409
-            code: CONFLICT
-            message: "The specified resource is in a conflict"
+          examples:
+            GENERIC_409_ABORTED:
```

Comment:
This case would be for two practically parallel requests. OK

Reply:
Yes - suppose we have 2 POST /Geofencing with same device, same area, same callbackUrl